libs/js-sdk/src/tests/http-client.spec.ts

```diff
@@ -341,13 +341,30 @@ describe('http client', () => {
                 updateStrategy: { kind: 'manual' },
             })
 
-            await httpClient.updateAudiences(['test-audience'])
+            await httpClient.waitForInitialised()
+
+            expect(httpClient.initialised).toBeTruthy
```

Comment:
This needs to be called `expect(httpClient.initialised).toBeTruthy()`

same with the ones below